ci/CI_test_list.yml

```diff
@@ -9,5 +10,3 @@ kube-burner:run_nodedensity-heavy_test_fromgit.sh
 storage-perf:run_storage_tests_fromgit.sh 
 upgrade-perf:run_upgrade_fromgit.sh 
 router-perf:run_router_test.sh 
```

Comment:
@rsevilla87 thoughts on removing the router-perf workload from the repo and replacing it with the v2?

Reply:
That's the idea, we can remove/replace it in another PR. (Didn't want to make any disrupting changes in this one)

Reply:
Sounds good.